### DISCUSSION TITLE
后端写什么？ | What are backend requirements?

### DISCUSSION THREAD

**Discussion by baiyuncode** [teaSummer/MCiSEE#39] [Ideas]
php

**Comment by teaSummer** [comment]
我之前在QQ上发了一个想法给你来着，不过还请优先处理 #2 。

**Comment by LateDreamXD** [comment]
我不到 反正大部分功能前端已经实现了XD
~这大部分都要归功于咱项目仓库的owner @teaSummer~

**Comment by baiyun1123** [comment]
我回来了

**Comment by baiyun1123** [comment]
emm

**Comment by baiyun1123** [comment]
最近在学习golang

**Comment by LateDreamXD** [comment]
awa

**Comment by teaSummer** [comment]
Give up backend supports for the moment.

But also you can see #37 and discuss more.

**Comment by LateDreamXD** [comment]
awa

**Comment by teaSummer** [comment]
Reopened, I assigned myself to finish this instead of @baiyun1123.

Now I have done the first plan. You can see [API_DOCS.md](https://github.com/teaSummer/MCiSEE/blob/main/API_DOCS.md) for detail.
